includes/discovery/vlans/vlan_functions.inc.php

```diff
@@ -1,37 +0,0 @@
-<?php
```

Comment:
@murrant I'm not sure of a good way to fix this... moving these functions into includes/functions.php breaks the unit tests when run locally (not sure why Travis isn't failing them)  But including the functions file in the unit test breaks the snmpsim stuff because of redefined functions... thoughts?